tests/x11/gnomeapps/aa_disable_updates.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Disable GNOME Software wanting to auto-update the system
+# Maintainer: Dominique Leuenberger <dimstar@suse.de>>
+
+use base "x11test";
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    x11_start_program 'xterm';
+    assert_screen 'xterm';
+    type_string "gsettings set org.gnome.software download-updates false\n";
```

Comment:
it would be - but then we can't get a screenshot of what it would have given as output (if any)

Normally there is nothing expected (hence also no asset screen) - but who knows. It might one day spit a stupid error and then it would be handy to have the screenshot available